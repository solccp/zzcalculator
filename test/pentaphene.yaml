---
structure_id: 1
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 21]
- [22, 19]
- [14, 13]
- [11, 6]
- [18, 17]
- [16, 9]
- [10, 5]
- [15, 8]
- [4, 7]
---
structure_id: 2
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 21]
- [22, 19]
- [14, 13]
- [11, 6]
- [18, 15]
- [17, 16]
- [9, 10]
- [8, 7]
- [5, 4]
---
structure_id: 3
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 21]
- [22, 19]
- [14, 13]
- [11, 6]
- [18, 15]
- [17, 16]
- [9, 8]
- [10, 5]
- [7, 4]
---
structure_id: 4
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 21]
- [22, 19]
- [14, 13]
- [11, 6]
- [18, 15]
- [17, 16]
rings:
- [9, 8, 7, 4, 5, 10]
---
structure_id: 5
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 21]
- [22, 19]
- [14, 13]
- [11, 6]
- [7, 4]
- [10, 5]
rings:
- [18, 15, 8, 9, 16, 17]
---
structure_id: 6
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 13]
- [21, 22]
- [11, 6]
- [19, 14]
- [17, 18]
- [16, 9]
- [10, 5]
- [4, 7]
- [8, 15]
---
structure_id: 7
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 13]
- [21, 22]
- [11, 6]
- [19, 14]
- [17, 16]
- [18, 15]
- [10, 9]
- [8, 7]
- [5, 4]
---
structure_id: 8
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 13]
- [21, 22]
- [11, 6]
- [19, 14]
- [17, 16]
- [18, 15]
- [10, 5]
- [4, 7]
- [8, 9]
---
structure_id: 9
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 13]
- [21, 22]
- [11, 6]
- [19, 14]
- [17, 16]
- [18, 15]
rings:
- [10, 5, 4, 7, 8, 9]
---
structure_id: 10
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [20, 13]
- [21, 22]
- [11, 6]
- [19, 14]
- [7, 4]
- [10, 5]
rings:
- [17, 16, 9, 8, 15, 18]
---
structure_id: 11
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [11, 6]
- [4, 7]
- [5, 10]
- [16, 17]
- [9, 8]
- [15, 18]
rings:
- [20, 13, 14, 19, 22, 21]
---
structure_id: 12
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [11, 6]
- [4, 7]
- [5, 10]
- [16, 9]
- [17, 18]
- [15, 8]
rings:
- [20, 13, 14, 19, 22, 21]
---
structure_id: 13
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [11, 6]
- [4, 7]
- [5, 10]
rings:
- [20, 13, 14, 19, 22, 21]
- [16, 9, 8, 15, 18, 17]
---
structure_id: 14
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [11, 6]
- [4, 5]
- [7, 8]
- [15, 18]
- [17, 16]
- [9, 10]
rings:
- [20, 13, 14, 19, 22, 21]
---
structure_id: 15
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 12]
- [11, 6]
- [15, 18]
- [17, 16]
rings:
- [20, 13, 14, 19, 22, 21]
- [4, 5, 10, 9, 8, 7]
---
structure_id: 16
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [20, 21]
- [22, 19]
- [12, 14]
- [16, 17]
- [18, 15]
- [9, 10]
- [8, 7]
- [5, 4]
---
structure_id: 17
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [20, 21]
- [22, 19]
- [12, 14]
- [16, 17]
- [18, 15]
- [9, 8]
- [10, 5]
- [7, 4]
---
structure_id: 18
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [20, 21]
- [22, 19]
- [12, 14]
- [16, 17]
- [18, 15]
rings:
- [9, 8, 7, 4, 5, 10]
---
structure_id: 19
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [20, 21]
- [22, 19]
- [12, 14]
- [16, 9]
- [17, 18]
- [15, 8]
- [10, 5]
- [7, 4]
---
structure_id: 20
number_of_atoms: 22
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [20, 21]
- [22, 19]
- [12, 14]
- [7, 4]
- [10, 5]
rings:
- [16, 9, 8, 15, 18, 17]
---
structure_id: 21
number_of_atoms: 22
double_bonds:
- [2, 3]
- [19, 22]
- [20, 21]
- [18, 17]
- [16, 9]
- [10, 5]
- [15, 8]
- [4, 7]
rings:
- [1, 6, 11, 13, 14, 12]
---
structure_id: 22
number_of_atoms: 22
double_bonds:
- [2, 3]
- [19, 22]
- [20, 21]
- [18, 15]
- [17, 16]
- [10, 9]
- [8, 7]
- [5, 4]
rings:
- [1, 6, 11, 13, 14, 12]
---
structure_id: 23
number_of_atoms: 22
double_bonds:
- [2, 3]
- [19, 22]
- [20, 21]
- [18, 15]
- [17, 16]
- [10, 5]
- [4, 7]
- [9, 8]
rings:
- [1, 6, 11, 13, 14, 12]
---
structure_id: 24
number_of_atoms: 22
double_bonds:
- [2, 3]
- [19, 22]
- [20, 21]
- [18, 15]
- [17, 16]
rings:
- [1, 6, 11, 13, 14, 12]
- [10, 5, 4, 7, 8, 9]
---
structure_id: 25
number_of_atoms: 22
double_bonds:
- [2, 3]
- [19, 22]
- [20, 21]
- [7, 4]
- [10, 5]
rings:
- [1, 6, 11, 13, 14, 12]
- [18, 15, 8, 9, 16, 17]
---
structure_id: 26
number_of_atoms: 22
double_bonds:
- [1, 2]
- [3, 4]
- [7, 8]
- [12, 14]
- [19, 22]
- [15, 18]
- [17, 16]
- [21, 20]
- [13, 11]
- [9, 10]
- [6, 5]
---
structure_id: 27
number_of_atoms: 22
double_bonds:
- [7, 8]
- [10, 9]
- [11, 13]
- [20, 21]
- [22, 19]
- [14, 12]
- [15, 18]
- [17, 16]
rings:
- [1, 2, 3, 4, 5, 6]
