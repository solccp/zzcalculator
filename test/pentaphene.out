 10 + 13 x + 4 x^2
 total: 27
